File: oss/src/main/java/com/truenine/component/oss/autoconfig/AutoConfigEntrance.java
package com.truenine.component.oss.autoconfig;


import com.truenine.component.oss.properties.AliCloudOssProperties;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Import;

/**
 * @author TrueNine
 * @since 2022-10-28
 */
@ComponentScan("io.tn.oss.autoconfig")
@Import(AliCloudOssProperties.class)
public class AutoConfigEntrance {
}

File: oss/src/main/kotlin/net/yan100/compose/oss/autoconfig/AutoConfigEntrance.kt
package net.yan100.compose.oss.autoconfig

import net.yan100.compose.oss.properties.AliCloudOssProperties
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Import

/**
 * @author TrueNine
 * @since 2022-10-28
 */
@ComponentScan("io.tn.oss.autoconfig")
@Import(AliCloudOssProperties::class)
class AutoConfigEntrance
